src/v3/src/transformer/testAttribute/transform.ts

```diff
@@ -81,7 +81,7 @@ const updateRegularButtons: TransformStepFn = (formbag) => {
     },
     callback: (element) => {
       const { options } = element as ButtonElement | LinkElement;
-      if (options.step) {
+      if (options.step && map[options.step]) {
```

Comment:
Fixes the issue with not being able to pass `dataSe` if the button had a `step` defined